### ISSUE TITLE
False positives for MD004 on lists inside blockquotes

### ISSUE THREAD

**Issue by toupeira** [markdownlint/markdownlint#261]
Markdown like this triggers MD004:

```
> - foo
> - bar
```

The problem seems to be in https://github.com/markdownlint/markdownlint/blob/a9e80fcf3989d73b654b00bb2225a00be53983e8/lib/mdl/doc.rb#L189 which looks at the first character of the line.

**Comment by jaymzh** [comment]
Fixed in 0.9.0.
